loadbalancer/etcd/test_certs/host.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you please include a README or something in this directory with clear, reproducible instructions on how to generate these files?